tests/function_corpus.py

```diff
@@ -689,6 +687,12 @@ def tensor_courpus_sum_dog(x, z, lhs_c, rhs_c, lhs_b, rhs_b):
 def tensor_courpus_add(x, y):
     return x + y
 
+def tensor_courpus_jacobian_add(x, y):
+    return x + y
+
+def tensor_courpus_jacobian_mul(x, y):
+    return x * y
+
```

Comment:
You should use tensor_courpus_add and tensor_courpus_jacobian_mul for this, no need for these